src/background/services/background.ts

```diff
@@ -179,6 +168,17 @@ export class Background {
         await this.storage.populate()
         await this.openPaymentsService.genererateKeys()
       }
+      this.checkPermissions()
```

Comment:
@raducristianpopa Should I bind permissions listener after install, or do you think safe enough to run in `Background.start()`?

Reply:
This should be safe enough.